recipes-bsp/libimxdmabuffer/libimxdmabuffer_1.1.2.bb

```diff
@@ -48,12 +53,12 @@ PACKAGECONFIG:append:mx8m-nxp-bsp     = " dwl"
 
 HANTRO_CONF = "--hantro-headers-path=${STAGING_INCDIR}/hantro_dec --hantro-decoder-version=G2"
 
-PACKAGECONFIG[dwl]      = "--with-dwl-allocator=yes ${HANTRO_CONF},--with-dwl-allocator=no,imx-vpu-hantro"
-PACKAGECONFIG[ion]      = "--with-ion-allocator=yes,               --with-ion-allocator=no,"
-PACKAGECONFIG[ipu]      = "--with-ipu-allocator=yes,               --with-ipu-allocator=no,"
-PACKAGECONFIG[g2d]      = "--with-g2d-allocator=yes,               --with-g2d-allocator=no,virtual/libg2d"
-PACKAGECONFIG[pxp]      = "--with-pxp-allocator=yes,               --with-pxp-allocator=no,"
-PACKAGECONFIG[dma-heap] = "--with-dma-heap-allocator=yes,          --with-dma-heap-allocator=no,"
+PACKAGECONFIG[dwl]      = "--with-dwl-allocator=yes ${HANTRO_CONF},        --with-dwl-allocator=no,imx-vpu-hantro"
+PACKAGECONFIG[ion]      = "--with-ion-allocator=yes,                       --with-ion-allocator=no,"
+PACKAGECONFIG[ipu]      = "--with-ipu-allocator=yes,                       --with-ipu-allocator=no,"
+PACKAGECONFIG[g2d]      = "--with-g2d-allocator=yes,                       --with-g2d-allocator=no,virtual/libg2d"
+PACKAGECONFIG[pxp]      = "--with-pxp-allocator=yes,                       --with-pxp-allocator=no,"
+PACKAGECONFIG[dma-heap] = "--with-dma-heap-allocator=yes ${DMA_HEAP_CONF}, --with-dma-heap-allocator=no,"
```

Comment:
Why you add a single instead one as: `dma-heap-cached` and `dma-heap-uncached`?

Reply:
Because there is one single dma-heap allocator. The values from `DMA_HEAP_CONF` are compile-time defaults. If users want to explicitly open a certain dma-heap, they specify an open dma-heap FD and whether that dma-heap is cached or not to `imx_dma_buffer_dma_heap_allocator_new_from_fd()`, otherwise they call `imx_dma_buffer_dma_heap_allocator_new()` and let libimxdmabuffer use the configured default.